forwarder_launch.py

```diff
@@ -88,6 +94,7 @@
 
     try:
         while True:
+            statistic_reporter.send_pv_numbers(len(update_handlers), time.time())
```

Comment:
Perhaps it would be better to use [RepeatTimer](https://github.com/ess-dmsc/forwarder/blob/master/forwarder/repeat_timer.py) instead of putting this in the consumer loop, but I think how you've done it is fine for an initial implementation that only reports number of PVs

Reply:
Yes. For this implementation we decided to use the consumer loop. In the next PR where we will send the total number of updates, we will use a thread.